modules/install/rke2_master.sh

```diff
@@ -96,7 +96,7 @@ sudo systemctl enable rke2-server
 sudo systemctl start rke2-server
 
```

Comment:
We could potentially get more status output in the log and better test efficiency by using something more like 

```
while ! systemctl is-active rke2-server; do sleep 2; done
```

Reply:
this lets wait a PR that is related to that to change , since its going to need more testing.